src/components/AboutModal.jsx

```diff
@@ -7,37 +7,30 @@ import {
 } from 'react-bootstrap';
 import cfgLogo from '../static/CfGLogoWhite.png';
 
-const AboutModal = (props) => {
-  const {
-    show,
-    hide,
-  } = props;
-
-  return (
-    <div>
-      <Modal show={show} onHide={hide}>
-        <Modal.Header closeButton>
-          <Modal.Title>About Code for Greensboro</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <h4>Code for Greensboro works to improve the way our community uses its data and the web.  And we need <strong>you</strong>—no coding required.</h4>
-          <br />
-          <p>We are a local 501(c)(3) organization affiliated with the Code for America Brigade program.
+const AboutModal = ({ show, hide }) => (
```

Comment:
Should we add some rules to our eslint to prefer destructuring?

Reply:
@SchaeStewart I would be for this. I always hate writing eslint rules / keeping them updated.. I ended up writing some configs for my work awhile back, might be worth looking into: https://github.com/rhinogram/eslint-config-rhinogram

Reply:
Okay cool. I'll dig into soon. I'm think we add rules for object destructuring and requring propTypes.